omnisafe/common/control_barrier_function/crabs/utils.py

```diff
@@ -15,12 +15,20 @@
 """Utils for CRABS."""
 # pylint: disable=all
 import os
+from typing import List, Union
```

Comment:
This typing style is not the same as other code. Currently we use the following style:
```python
devices: Union[List[int], int] # not used
devices: list[int]|int # currently used
```
To use the second style, please add one line on the top of the file:
```python
from __future__ import annotations
```